misc/python/materialize/git.py

```diff
@@ -352,8 +352,10 @@ def tag_annotated(tag: str) -> None:
     spawn.runv(["git", "tag", "-a", "-m", tag, tag])
 
 
-def push(remote: str, remote_ref: str | None = None) -> None:
+def push(remote: str, remote_ref: str | None = None, tag: str | None = None) -> None:
     if remote_ref:
         spawn.runv(["git", "push", remote, f"HEAD:{remote_ref}"])
+    elif tag:
+        spawn.runv(["git", "push", remote, tag])
```

Comment:
This is pretty ugly, so I'm open to suggestions.

Reply:
I don't think this function is called anywhere else? I'd just remove it, and inline the call to `spawn.runv` that you need in `cut-release.py`.

Reply:
Done